### PR TITLE
Add Roundup Issue Tracker to the Ticketing section.

### LINKED ISSUE #2859: Add Roundup Issue Tracker to the Ticketing section
The Pull request is at: https://github.com/awesome-selfhosted/awesome-selfhosted/pull/2855.
The Roundup Issue Tracker is celebrating its 20th anniversary. It has been the tracking software for bugs.python.org since 2006. Please consider adding this software under active development to your list.

Thanks.
-- rouilj

Thank you for taking the time to work on a PR for Awesome-Selfhosted!

To ensure your PR is dealt with swiftly please check the following:

- [X] Submit one item per pull request. This eases reviewing and speeds up inclusion.
- [X] Format your submission as follows, where `Demo` and `Clients` are optional.
  Do not add a duplicate `Source code` link if it is the same as the main link.
  Keep the short description under 250 characters and use [sentence case](https://en.wikipedia.org/wiki/Letter_case#Sentence_case)
  for it, even if the project's webpage or readme uses another capitalisation
  such as title case, all caps, small caps or all lowercase.
  `Demo` links should only be used for interactive demos, i.e. not video demonstrations.
  ``- [Name](http://homepage/) - Short description, under 250 characters, sentence case. ([Demo](http://url.to/demo), [Source Code](http://url.of/source/code), [Clients](https://url.to/list/of/related/clients-or-apps)) `License` `Language` ``
- [X] Additions that depend on proprietary services outside the user's control must be marked `⚠`.
  ``- [Name](http://homepage/) `⚠` - Short description, under 250 characters, sentence case. ([Demo](http://url.to/demo), [Source Code](http://url.of/source/code), [Clients](https://url.to/list/of/related/clients-or-apps)) `License` `Language` ``
- [X] Additions that are not [Free software](https://en.wikipedia.org/wiki/Free_software)
  must be added to `non-free.md` and marked `⊘ Proprietary`:
  ``- [Name](http://homepage/) `⊘ Proprietary` - Short description, under 250 characters, sentence case. ([Demo](http://url.to/demo), [Source Code](http://url.of/source/code), [Clients](https://url.to/list/of/related/clients-or-apps)) `Language` ``
- [X] Additions are inserted preserving alphabetical order.
- [X] Additions are not already listed at any of [awesome-sysadmin](https://github.com/n1trux/awesome-sysadmin), [awesome-analytics](https://github.com/onurakpolat/awesome-analytics), [staticgen.com](https://www.staticgen.com/), [staticsitegenerators.net](https://staticsitegenerators.net/).
- [X] The `Language` tag is the main **server-side** requirement for the software. Don't include frameworks or specific dialects.
- [X] Any license you add is in our [list of licenses](https://github.com/awesome-selfhosted/awesome-selfhosted/blob/master/README.md#list-of-licenses).
- [X] You have searched the repository for any relevant [issues](https://github.com/awesome-selfhosted/awesome-selfhosted/issues) or [PRs](https://github.com/awesome-selfhosted/awesome-selfhosted/pulls), including closed ones.
- [X] Any category you are creating has the minimum requirement of 3 items.
  If not, your addition may be inserted into `Misc/Other`.
- [X] Any software project you are adding to the list is actively maintained.
- [X] The pull request title is informative, unlike "Update README.md".
  Suggested titles: "Add aaa to bbb" for adding software aaa to section bbb,
  "Remove aaa from bbb" for removing, "Fix license for aaa", etc.

### PR DESCRIPTION
**Pull request by rouilj**
The Roundup Issue Tracker is celebrating its 20th anniversary. It has been the tracking software for bugs.python.org since 2006. Please consider adding this software under active development to your list.

Fixes #2859.

Thanks.
-- rouilj

Thank you for taking the time to work on a PR for Awesome-Selfhosted!

To ensure your PR is dealt with swiftly please check the following:

- [X] Submit one item per pull request. This eases reviewing and speeds up inclusion.
- [X] Format your submission as follows, where `Demo` and `Clients` are optional.
  Do not add a duplicate `Source code` link if it is the same as the main link.
  Keep the short description under 250 characters and use [sentence case](https://en.wikipedia.org/wiki/Letter_case#Sentence_case)
  for it, even if the project's webpage or readme uses another capitalisation
  such as title case, all caps, small caps or all lowercase.
  `Demo` links should only be used for interactive demos, i.e. not video demonstrations.
  ``- [Name](http://homepage/) - Short description, under 250 characters, sentence case. ([Demo](http://url.to/demo), [Source Code](http://url.of/source/code), [Clients](https://url.to/list/of/related/clients-or-apps)) `License` `Language` ``
- [X] Additions that depend on proprietary services outside the user's control must be marked `⚠`.
  ``- [Name](http://homepage/) `⚠` - Short description, under 250 characters, sentence case. ([Demo](http://url.to/demo), [Source Code](http://url.of/source/code), [Clients](https://url.to/list/of/related/clients-or-apps)) `License` `Language` ``
- [X] Additions that are not [Free software](https://en.wikipedia.org/wiki/Free_software)
  must be added to `non-free.md` and marked `⊘ Proprietary`:
  ``- [Name](http://homepage/) `⊘ Proprietary` - Short description, under 250 characters, sentence case. ([Demo](http://url.to/demo), [Source Code](http://url.of/source/code), [Clients](https://url.to/list/of/related/clients-or-apps)) `Language` ``
- [X] Additions are inserted preserving alphabetical order.
- [X] Additions are not already listed at any of [awesome-sysadmin](https://github.com/n1trux/awesome-sysadmin), [awesome-analytics](https://github.com/onurakpolat/awesome-analytics), [staticgen.com](https://www.staticgen.com/), [staticsitegenerators.net](https://staticsitegenerators.net/).
- [X] The `Language` tag is the main **server-side** requirement for the software. Don't include frameworks or specific dialects.
- [X] Any license you add is in our [list of licenses](https://github.com/awesome-selfhosted/awesome-selfhosted/blob/master/README.md#list-of-licenses).
- [X] You have searched the repository for any relevant [issues](https://github.com/awesome-selfhosted/awesome-selfhosted/issues) or [PRs](https://github.com/awesome-selfhosted/awesome-selfhosted/pulls), including closed ones.
- [X] Any category you are creating has the minimum requirement of 3 items.
  If not, your addition may be inserted into `Misc/Other`.
- [X] Any software project you are adding to the list is actively maintained.
- [X] The pull request title is informative, unlike "Update README.md".
  Suggested titles: "Add aaa to bbb" for adding software aaa to section bbb,
  "Remove aaa from bbb" for removing, "Fix license for aaa", etc.

### CODE REVIEW

**Comment by rouilj** [comment]
@nodiscc I see you added a "feedback needed" label three days ago. But I see no information on what feedback you need on the pull request or its linked issue.

**Comment by Epy** [comment]
Hello
Link in HTTPS doesn't seem to be working, in HTTP I get a Cloudflare 1001 error

**Comment by rouilj** [comment]
Just an update, <https://www.roundup-tracker.org/> should be working again.

File: README.md [hunk 1]
```diff
@@ -1798,6 +1798,7 @@ _Related: [Task management & To-do lists](#task-management--to-do-lists), [Softw
 - [osTicket](https://osticket.com/) - Manage, organize and archive all your support requests and responses in one place. ([Source Code](https://github.com/osTicket/osTicket)) `GPL-2.0` `PHP`
 - [Pachno](https://pach.no/) - Bring your team together to design, build and deliver your project with a tool that works with you and your team, and adapts when you need to. ([Source Code](https://github.com/pachno/pachno)) `MPL-2.0` `PHP`
 - [Request Tracker](https://www.bestpractical.com/rt/) - An enterprise-grade issue tracking system. ([Source Code](https://github.com/bestpractical/rt)) `GPL-2.0` `Perl`
+- [Roundup Issue Tracker](https://www.roundup-tracker.org/) - A simple-to-use and -install issue-tracking system with command-line, web, REST, XML-RPC, and e-mail interfaces. Designed with flexibility in mind - not just another bug tracker. Unpack source, run “python demo.py”, open the printed demo URL!  ([Source Code](https://www.roundup-tracker.org/code.html)) `MIT, ZPL-2.0` `Python3 or Python2`
```
Comment (nodiscc) [inline README.md:1801]: please use the official description from https://www.roundup-tracker.org/

**Review by nodiscc — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (nodiscc) [reply README.md:1801]: - Please change the license tag to `MIT/ZPL-2.0`
- Please change the language tag to `Python`

Comment (rouilj) [reply README.md]: > please use the official description from https://www.roundup-tracker.org/

My description is almost word for word pulled from the site. The relevant
paragraphs are:

paragraph 1
```
Roundup is a simple-to-use and -install issue-tracking system with command-line, web and e-mail interfaces. It is based on the winning design from Ka-Ping Yee in the Software Carpentry “Track” design competition.
```
I added/highlighted the REST interface (mentioned in paragraph 2 of the README and at <https://www.roundup-tracker.org/docs/rest.html>) and the older xmlrpc (<https://www.roundup-tracker.org/docs/xmlrpc.html>) interface in the leading line to save space.

Paragraph 6 says:
```
…and so on. It’s been designed with [flexibility](https://www.roundup-tracker.org/docs/customizing.html) in mind - it’s not just another bug tracker. Roundup ships with a demo tracker to play with - after you’ve unpacked the source, just run “python demo.py” and load up the URL it prints out!
```
which was edited for brevity and the link dropped.

I think these are the important points for a short description and similar to other descriptions and announcements I have done. It:

* describes the software and how you can interact with it
* hints at its additional capabilities for those interested
* describes how to get a taste of Roundup using demo mode

Can you be more specific as to your concerns about my wording of the short description?

Also note that I will change the first line of the web site to more closely match the wording here. Historically we didn't highlight the API interfaces.

Comment (nodiscc) [reply README.md]: > Can you be more specific as to your concerns about my wording of the short description?

Adding installation instructions `Unpack source, run “python demo.py”, open the printed demo URL!` to the description is going a bit far :) And descriptions must end with a `.`

Removing just this sentence would be fine

File: README.md [hunk 2]
```diff
@@ -1798,6 +1798,7 @@ _Related: [Task management & To-do lists](#task-management--to-do-lists), [Softw
 - [osTicket](https://osticket.com/) - Manage, organize and archive all your support requests and responses in one place. ([Source Code](https://github.com/osTicket/osTicket)) `GPL-2.0` `PHP`
 - [Pachno](https://pach.no/) - Bring your team together to design, build and deliver your project with a tool that works with you and your team, and adapts when you need to. ([Source Code](https://github.com/pachno/pachno)) `MPL-2.0` `PHP`
 - [Request Tracker](https://www.bestpractical.com/rt/) - An enterprise-grade issue tracking system. ([Source Code](https://github.com/bestpractical/rt)) `GPL-2.0` `Perl`
+- [Roundup Issue Tracker](https://www.roundup-tracker.org/) - A simple-to-use and -install issue-tracking system with command-line, web, REST, XML-RPC, and e-mail interfaces. Designed with flexibility in mind - not just another bug tracker.  ([Source Code](https://www.roundup-tracker.org/code.html)) `MIT/ZPL-2.0` `Python`
```
Comment (nodiscc) [inline README.md:1801]: Double space `tracker. ([Source Code`

**Review by nodiscc — CHANGES_REQUESTED** [review]

**Review by nodiscc — APPROVED** [review]
Thanks

**Pull request merged**